standalone/instructlab-rhoai.md

```diff
@@ -81,6 +81,7 @@ Before running the training and evaluation steps we must complete the following:
 
 
 ### Step 2 - Create Judge model server
```

Comment:
```suggestion
### Step 2 (Optional) - Create Judge model server
```

Should we do this to maintain consistency with the other optional sections?

Reply:
@leseb do you think including this in the title will mean that the user might miss the requirement to create the secret?

Reply:
You're right. I did not realize this was a subsection. Let's not complicate things then.